index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <h1 id="title">Dr. Norman Bourlag</h1>
+      <p>The man who saved a billion lives</p>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="Dr. Normat talking"
+        />
+        <figcaption id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </figcaption>
+      </div>
```

Comment:
👏Great Job!!

- [ ] I would suggest the following:

✨ Please consider wrapping these tags with a header tag. A semantic element clearly describes its meaning to the browser and the developer. These are some examples of semantic elements: <form>, <table>, <article>, <header>, and <footer>. In this section, you can place the main information about the site, site identification, phrases, or slogans. 👨‍🚀 This is beneficial for the SEO positioning of the site online because it favors its indexing by search engines such as Google. Likewise, a well-structured header can better convey the main information to the user. 👍

🔍 I recommend the following resource: https://www.w3schools.com/html/html5_semantic_elements.asp